components/brave_rewards/resources/extension/brave_rewards/components/panel.tsx

```diff
@@ -331,6 +331,46 @@ export class Panel extends React.Component<Props, State> {
     }
   }
 
+  onContributionAmountChange = (publisherKey: string, newAmount: string) => {
+    const newValue = parseInt(newAmount, 10)
```

Comment:
For Typescript, I think use:
`const newValue = number(newAmount)`
but I don't know if there's a convention. wdyt?

Reply:
We generally have been sticking to `parseInt` throught the brave rewards client side so I think we should stick with it. If there's some other benefit of `number` we are missing out on we can discuss switching over 👍

Reply:
If that's what's been used then I would stick with it.